### DISCUSSION TITLE
Can't push "ArtifactDb" and "SourceAssetDB"

### DISCUSSION THREAD

**Discussion by fadedmemory1500** [git-lfs/git-lfs#4854] [Q&A]
I've recently installed Vuforia 10.4.4 to my unity 2020.3.21f.

after I initiated github project, and git LFS, I realized I can't push these two files to GitHub desktop anymore. 
Library\ ArtifactDb
Library\ SourceAssetDB
github desktop say it exceed the 100mb limit, but that's why I use gitLFS.

P.S.
I did all   git lfs track ".filetype"  tracking command line for all file types
I had .gitignore in place
![image](https://user-images.githubusercontent.com/82472982/152421160-991870fc-f0d9-4c20-848b-4634a95ad77d.png)

**Comment by bk2204** [comment]
Hey,

Can you provide your `.gitattributes` file and the message you're seeing from GitHub with the specific files?  We may be able to offer a solution to help out with this.

**Reply by fadedmemory1500** [reply]
here is the code inside the .gitattributes:

.file filter=lfs diff=lfs merge=lfs -text
.psd filter=lfs diff=lfs merge=lfs -text
.pnggit filter=lfs diff=lfs merge=lfs -text
lfs filter=lfs diff=lfs merge=lfs -text
track filter=lfs diff=lfs merge=lfs -text
.png filter=lfs diff=lfs merge=lfs -text
.ogg filter=lfs diff=lfs merge=lfs -text
.tgz filter=lfs diff=lfs merge=lfs -text
dir/*
git[[:space:]]lfs[[:space:]]track[[:space:]]dir/library* filter=lfs diff=lfs merge=lfs -text

**Reply by fadedmemory1500** [reply]
here is the error message in the github desktop when I'm trying to push: 
![image](https://user-images.githubusercontent.com/82472982/152434314-10292a3f-66b3-429e-840d-ad059a457216.png)

**Reply by bk2204** [reply]
Hey,

Here's what it looks like is going wrong.

First, the pattern `.psd` matches literally the file `.psd` anywhere in the tree.  It doesn't match all files ending in `.psd`; for that, you want `*.psd`.  Second, it looks like you have some junk in the file: you shouldn't have `.pnggit`, `lfs`, or `track` there.  Third, Git is case sensitive, so if your directory is `Library`, then the pattern you want is `Library/*`.

If you haven't already committed these files, then I'd recommend editing your `.gitattributes` to look like this:

```
*.file filter=lfs diff=lfs merge=lfs -text
*.psd filter=lfs diff=lfs merge=lfs -text
*.png filter=lfs diff=lfs merge=lfs -text
*.ogg filter=lfs diff=lfs merge=lfs -text
*.tgz filter=lfs diff=lfs merge=lfs -text
Library/* filter=lfs diff=lfs merge=lfs -text
```

Then, run `git add --renormalize .` and commit the files.  If you have already committed them, then make sure you have a clean working tree, and then run `git lfs migrate import --everything --include="*.file,*.psd,*.png,*.ogg,*.tgz,Library/*"`, which should fix things.

**Reply by fadedmemory1500** [reply]
thanks a lot. it seems working now.

**Reply by Tengku3077** [reply]
Google drive
